modules/virt-importing-vm-datavolume.adoc

```diff
@@ -49,7 +50,7 @@ $ oc apply -f endpoint-secret.yaml
 ----
```

Comment:
Regarding base64 encoding, can we add a caution about accidentally encoding newlines?

(
We had this come up in OpenShift Virtualization google chat group:
Alexander Wels, Oct 15, 8:04 PM
Like I said | base64 also encodes the \n and I think you probably the 4th or 5th person I know that got bit by that.
)

Reply:
Very interesting! I've looked into it and I'm not really sure this fits the scope of this doc though. From what I understand it's dependent on the implementation of base64 as to whether that linewrap is default behaviour, and even then there's ways around it. If we included the encoding part in this procedure then it would make sense to include a note here, but we're just asking for base64-encoded credentials in the config - how the user goes about providing that is really up to them.
But I appreciate the suggestion, and will probably spend the rest of the day thinking about this.